src/engine/server/sv_init.cpp

```diff
@@ -52,6 +52,9 @@ static Cvar::Cvar<std::string> cvar_pakname(
 	"pakname", "pak containing current map", Cvar::SERVERINFO | Cvar::ROM, "");
 static Cvar::Cvar<std::string> sv_paks(
 	"sv_paks", "currently loaded paks", Cvar::SYSTEMINFO | Cvar::ROM, "");
+static Cvar::Cvar<std::string> cvar_abuse(
```

Comment:
Why call it "abuse" if that's not the main motivation for providing it? How about "contact" or "admin"

Reply:
This can be named `contact`, I first named it `admin` but then it sounded weird because there are other `admin*` commands and this would be confusing. Then my next example in mind was the `abuse@` mail address convention. But I'm fully open to other names, like `contact`.

Reply:
I renamed to `contact`.